scripts/python/startSwarm.py

```diff
@@ -45,6 +45,21 @@
         node.runCmdThread(command)
 
     time.sleep(15)
+
+    print "Check netplugin is installed and enabled"
+    out, x, y = nodes[0].runCmd("docker plugin ls")
```

Comment:
if you don't care about the err and exit value, use '_', though I think you could have docker installed but docker plugin ls fail when dockerd is not running, so you might want to check exit status

Reply:
👍 done.